freqtrade/data/btanalysis.py

```diff
@@ -168,6 +170,14 @@ def find_existing_backtest_stats(dirname: Union[Path, str],
             # is safe to assume older files will also not have any metadata.
             break
 
+        if min_backtest_date is not None:
+            backtest_date = datetime.strptime(
+                f'{filename.name}', 'backtest-result-%Y-%m-%d_%H-%M-%S.json')
```

Comment:
I'm not sure how you imagine this to work (i don't think we can use the filename).

assume the following:
i use `backtesting` yesterday - and continue today with  `backtesting --cache week`.
It'll load yesterday's results - and store them into today's file (so far, so good).

I'm now using `--cache day` - so i'd expect only results that were _backtested_ today to be considered.
The one from yesterday however got "moved over" to today's file - so it'll pass this filter (i can move a backtest result over weeks/months this way).


I think we'll need to extend the metadata file with the "backtest-date" - storing when the original backtest was made (and obviously move this date unchanged to new metadata files).

Reply:
Gah well that is a very good point 😬